### DISCUSSION TITLE
ClickHouse GUI

### DISCUSSION THREAD

**Discussion by jdjohn555** [ClickHouse/ClickHouse#54444] [Show and tell]
Hi,

I recently added support for ClickHouse to qStudio.
qStudio is a free SQL GUI, for running SQL scripts, browsing tables and charting results.
https://www.timestored.com/qstudio/database/clickhouse
We would like feedback from someone more experienced with ClickHouse.
If you are looking for a GUI, please give it a try and if you have any questions or feature requests
please raise an issue at:
https://github.com/timeseries/qstudioopen
![image](https://github.com/ClickHouse/ClickHouse/assets/141037965/f433a758-4a07-4f4c-9a87-3caaeff4c8f3)

**Comment by den-crane** [comment]
tried 3 times, failed

gmail.com
Registration Successful

Your license key is:

```
H4sIAAAAAAAAAGNgYNAz0GEScdI2imJZdcGoZc3MQxYeF+6kanKIMYnExzU/s3NQkMh/NU1L5JhpociK
HQk2OZnJqXnFqQr55XmpRbZKiYmJesXFDum5iZk5esn5uUp2XAoKNgVF+SmlySU+ELUgIYSgQklmSU6q
rVJhcElpSma+kkJxSWJRiUtiCVDMyMDIWNfAUtfAXMHIwMrQ3MrUSM/I1EghNMRZSSE1LwWuygSkytAI
UxXELqBtaamJJaVFqVAnFOtDHaEPdQXYmfro7rTRh3rPDgCxq8dSGgEAAA==
```

<img width="767" alt="Screenshot 2023-09-08 at 5 18 17 PM" src="https://github.com/ClickHouse/ClickHouse/assets/19737682/bd75e2b6-63ed-41b8-9867-e683a370fb1a">

**Comment by ryanhamilton** [comment]
Really sorry to see this happened you Den. Here's a key that will work for anyone until August 2024.

```
QSV3KH4sIAAAAAAAAAGNgYNAz0GESyTkZE5CzaqGu8wql5dZvC55f/mskyCRi2ikc+HePqLyI8jr7+lf
HDqVt/7fXJiczOTWvOFUhvzwvtchWKbUstagyPy/VISuxLNHQXC85P1fJjktBwaagKD+lNLnEB6IcJIQ
QVCjJLMlJtVUqDC4pTcnMV1IoLkksKnFJLAGKGRkYGesaWOgamigYWlgZG1uZWOiZGpophIY4Kymk5qX
AVZmAVVliqoLYBbQtLTWxpLQoFeqEYn2oI/ShrgA7Ux/dnTb6UB/aAQBCLxbMHQEAAA==
```

We have done the work to make qStudio free, we have started to open source it, eventually we want to provide a fully open build and release for download.
That includes removing the old key registration that relies on pre java-17.
Thanks for giving it a try and by raising this issue it raises that work up our priority list.
